### PR TITLE
Rtomlinson latacora feature/120 gcp svc act multi proj

### PR DESCRIPTION
**Pull request by j4v**
Made a few tweaks to https://github.com/nccgroup/ScoutSuite/pull/162, added some exception handling and resolved conflicts with `develop`.

### CODE REVIEW

**Comment by j4v** [comment]
@rtomlinson-latacora had to do some git magic to resolve conflicts, your changes are now in this PR. Looks good to you?

File: ScoutSuite/providers/gcp/provider.py [hunk 1]
```diff
@@ -58,7 +59,7 @@ def authenticate(self, user_account=None, service_account=None, **kargs):
             warnings.filterwarnings("ignore", "Your application has authenticated using end user credentials")
             pass  # Nothing more to do
         elif service_account:
-            client_secrets_path = os.path.abspath(service_account)  # TODO this is probably wrong
+            client_secrets_path = os.path.abspath(service_account)
```
Comment (Aboisier) [inline ScoutSuite/providers/gcp/provider.py]: 🥇

**Comment by rtomlinson-latacora** [comment]
Looks good! Thanks again

**Comment by j4v** [comment]
> Looks good! Thanks again

Merging, thanks for your help and sorry for the delay!

**Pull request merged**